src/screens/new-product/Layout.jsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { useState } from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import GoBackButton from '../../components/GoBackButton';
+import './style.css'
+import CreateProductButton from '../../components/CreateProductButton';
+import { addProducts } from '../../services/addProductService';
+
+
+export default function NewProductForm() {
+
+  const [inputs, setInputs] = useState({})
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setInputs(values => ({...values, [name]: value}))
+  }
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const result = await addProducts(inputs);
+      console.log(result)
+      alert('Producto creado con éxito');
+      window.location.href = '/products'
```

Comment:
Use react-router to navigate, don't use `window.location.href` never.
This method reloads all app.